control-plane/stor-port/src/types/v0/transport/replica.rs

```diff
@@ -105,23 +105,23 @@ pub struct CreateReplicaSnapshotResp {
 /// A single snapshot descriptor.
 pub struct ReplicaSnapshotDescr {
     /// UUID of the snapshot.
-    snap_uuid: SnapshotId,
+    pub snap_uuid: SnapshotId,
```

Comment:
you can add getter methods rather than make these pub

Reply:
You'd need a new() method to take them all, I'll leave it up to you to determine which is better here.